packages/angular-generator/src/expressions/class-members/get-accessor.ts

```diff
@@ -21,6 +21,11 @@ export class GetAccessor extends BaseGetAccessor {
   }
 
   toString(options?: toStringOptions): string {
+    if (this.name.toString() === '__restAttributes') {
```

Comment:
Seems it can be moved to the getter because used multiple times.
```
get isRestAttributes() {
 return this.name.toString() === '__restAttributes';
}
```